sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/statsbeat/Feature.java

```diff
@@ -68,7 +68,9 @@ public enum Feature {
     PREVIEW_DONT_USE_NORMALIZED(54), // track useNormalizedValueForNonNormalizedCpuPercentage
     PREVIEW_CUSTOM_INSTRUMENTATION(55), // track customInstrumentation
     PREVIEW_ADDITIONAL_PROPAGATORS(56), // track additionalPropagators
-    GRAAL_VM_NATIVE(57);
+    GRAAL_VM_NATIVE(57),
+    PEKKO_DISABLED(58) // preview instrumentation
```

Comment:
in Java agent, you would need to update initStatsbeatFeatureSet in order to start tracking this feature.